setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup, find_packages
+
+# read the contents of your README file
+from os import path

```

Comment:
Please keep imports together on the top, separated by a blank line from the rest:
```suggestion
from setuptools import setup, find_packages
from os import path

# read the contents of your README file
```